catalog/dags/data_refresh/create_filtered_index_dag.py

```diff
@@ -218,13 +220,19 @@ def point_alias(destination_index_suffix: str):
 
         (
             prevent_concurrency
-            >> destination_index_suffix
-            >> get_current_index
+            >> [destination_index_suffix, get_current_index]
+            # Gather step which will run even if the get current index step is skipped
+            >> EmptyOperator(
+                task_id="pre_create_check",
+                trigger_rule=TriggerRule.NONE_FAILED_MIN_ONE_SUCCESS,
+            )
```

Comment:
Because these two lines are kind of their own unit, would you mind moving them into a helper function or their own variable with a clearer explanatory comment? Something along the lines of "`get_current_index` can fail if the filtered index does not exist (i.e., on first runs) so we'll only require `destination_index_suffix` to succeed in order to continue with the DAG" would be very helpful. It might just be due to my lack of Airflow experience but putting the `EmptyOperator` inline doesn't make the relationship clear between it and the previous group. A change to the task ID to relate it to the task group in question (rather than the subsequent task) would also help with clarity.

Reply:
Sure! I'll see if there's a better way I can aggregate/separate them so it's clearer what's going on here.

Reply:
Okay, I've reworked this a bit in 9d62ac9030e700d7435069d0397f9450c8908d6c, let me know if that's clearer! Also, if you have any suggestions for the task ID or the name for improved clarity, I'm open to them 🙂 I've also updated the PR description to reflect the new DAG.